### PR TITLE
get rid of g_localizeStrings in Dialogs

### PR DESCRIPTION
**Pull request by MartijnKaijser**
this one is based on #7200 as such only second commit counts

### CODE REVIEW

**Comment by MartijnKaijser** [comment]
@xhaggi @da-anda

File: xbmc/video/windows/GUIWindowVideoBase.cpp [hunk 1]
```diff
@@ -1883,12 +1883,12 @@ void CGUIWindowVideoBase::AppendAndClearSearchItems(CFileItemList &searchItems,
...
 }
 
-bool CGUIWindowVideoBase::OnUnAssignContent(const std::string &path, int label1, int label2, int label3)
+bool CGUIWindowVideoBase::OnUnAssignContent(const std::string &path, int header, int text)
 {
   bool bCanceled;
   CVideoDatabase db;
   db.Open();
-  if (CGUIDialogYesNo::ShowAndGetInput(label1, label2, label3, "", bCanceled))
+  if (CGUIDialogYesNo::ShowAndGetInput(header, text, bCanceled, "", ""))
```
Comment (xhaggi) [inline xbmc/video/windows/GUIWindowVideoBase.cpp]: no need for the last 2 empty strings here, there are defaults for.

File: xbmc/video/windows/GUIWindowVideoBase.cpp [hunk 2]
```diff
@@ -1926,11 +1926,11 @@ void CGUIWindowVideoBase::OnAssignContent(const std::string &path)
   {
     if(settings.exclude || (!info && info2))
     {
-      OnUnAssignContent(path,20375,20340,20341);
+      OnUnAssignContent(path,20375,20340);
```
Comment (xhaggi) [inline xbmc/video/windows/GUIWindowVideoBase.cpp]: minor, spaces between parameter

**Comment by xhaggi** [comment]
apart from my comments looks good

File: addons/resource.language.en_gb/resources/strings.po [hunk 3]
```diff
@@ -3407,21 +3406,12 @@ msgctxt "#1001"
...
 #: xbmc/dialogs/GUIDialogMediaSource.cpp
-msgctxt "#1003"
-msgid "This could be due to the network not being connected."
-msgstr ""
-
-#: xbmc/dialogs/GUIDialogMediaSource.cpp
-msgctxt "#1004"
-msgid "Would you like to add it anyway?"
+msgctxt "#1002"
+msgid "The connection to the network location couldn't be established. This could be due to the network not being connected. Would you like to add it anyway"
```
Comment (da-anda) [inline addons/resource.language.en_gb/resources/strings.po]: Missing question mark at end

File: xbmc/GUIPassword.cpp [hunk 4]
```diff
@@ -143,7 +143,7 @@ bool CGUIPassword::CheckStartUpLock()
         std::string strLabel = StringUtils::Format("%i %s", iLeft, strLabel1.c_str());
 
         // PopUp OK and Display: MasterLock mode has changed but no new Mastercode has been set!
-        CGUIDialogOK::ShowAndGetInput(12360, 12367, 12368, strLabel);
+        CGUIDialogOK::ShowAndGetInput(12360, 12368, strLabel, "");
```
Comment (da-anda) [inline xbmc/GUIPassword.cpp]: 12368 has to be 12367

File: xbmc/pvr/addons/PVRClients.cpp [hunk 5]
```diff
@@ -1137,7 +1137,7 @@ bool CPVRClients::UpdateAndInitialiseClients(bool bInitialiseAllClients /* = fal
       }
 
       if (bDisabled && (g_PVRManager.IsStarted() || g_PVRManager.IsInitialising()))
-        CGUIDialogOK::ShowAndGetInput(24070, 24071, 16029, 0);
+        CGUIDialogOK::ShowAndGetInput(24070, 16029);
```
Comment (da-anda) [inline xbmc/pvr/addons/PVRClients.cpp]: Is it by intention that you dropped the 24071 ("An unknown error has occurred.") part? IMO it's fine to be dropped, just want to be sure it was also your intention and not by accident.

**Comment by da-anda** [comment]
looks good from my end once comments are addressed

File: xbmc/pvr/addons/PVRClients.cpp (hunk 5 above)
Comment (MartijnKaijser) [inline xbmc/pvr/addons/PVRClients.cpp]: yep by intention

**Comment by MartijnKaijser** [comment]
jenkins build and merge

**Pull request merged**